.travis-ipython4-requirements.txt

```diff
@@ -0,0 +1,22 @@
+Jinja2>=2.7.3
+Mako>=1.0.0
+MarkupSafe>=0.23
+Pygments>=2.0.1
+SQLAlchemy>=1.0.5
+alembic>=0.7.6
+backports.ssl-match-hostname>=3.4.0.2
+certifi>=14.05.14
+ipython>=4.0.0
```

Comment:
I added here traitlets, notebook and nbformat. These files are only needed for IPython 4.0 and break the IPython 3.2 tests.